src/cr/cube/matrix.py

```diff
@@ -2646,7 +2651,7 @@ def zscore(self):
         return self._zscore
 
 
-class _MultipleResponseVector(_CategoricalVector):
+class _OpposingMrVector(_CategoricalVector):
     """CAT or MR vector that opposes an MR dimension.
```

Comment:
The commit message for this one is _too_ valuable and well-written to allow it to be lost in `git` history. Please include it in the docstring of the class ❤️

Reply:
Also, as for the other class, the word "orthogonal" just won't leave me alone. I think it describes our use case better than "opposing", which might mean anything.